explore/src/main/scala/explore/tabs/ObsTabTiles.scala

```diff
@@ -207,8 +197,8 @@ object ObsTabTiles {
           obsView.toOption.map(_.get.scienceData.constraints),
           obsView.toOption.flatMap(_.get.scienceData.requirements.spectroscopy.wavelength),
           props.focusedTarget,
-          Reuse(setCurrentTarget _)(props.programId, props.focusedObs),
-          Reuse.currying(props.targetMap, props.obsId).in(otherObsCount _),
+          setCurrentTarget(props.programId, props.focusedObs),
+          otherObsCount(props.targetMap, props.obsId, _),
```

Comment:
A little leftover reuse, eh? Good catch.

Reply:
Yeah, I bet there are few more here and there